x-pack/plugins/apm/server/lib/transactions/charts/get_timeseries_data/fetcher.ts

```diff
@@ -139,11 +136,5 @@ export function timeseriesFetcher({
     }
   };
 
-  if (transactionName) {
-    params.body.query.bool.must = [
-      { term: { [TRANSACTION_NAME]: transactionName } }
-    ];
-  }
```

Comment:
No reason to have a must when we can simply add `transactionName` to the filter like `transactionType`